images/diagnostics/Dockerfile

```diff
@@ -0,0 +1,9 @@
+#
+# OpenShift diagnostics image
+# Used by network diagnostics (oadm diagnostics NetworkCheck)
+#
+# The standard name for this image is openshift/diagnostics-deployer
+
+FROM docker.io/busybox
```

Comment:
Main purpose of using busybox is to keep the image size as small as possible since this will be pulled on all nodes for diagnostics command. Any specific reason for switching to rhel7 base image?